packagebuild/workflows/build_package.wf.json

```diff
@@ -60,6 +63,7 @@
             "repo-owner": "${repo_owner}",
             "repo-name": "${repo_name}",
             "git-ref": "${git_ref}",
+            "extra-git-ref": "${extra_git_ref}",
```

Comment:
We should modify the `daisy_startupscript-XXX.sh` scripts to also use these `refs`.

Reply:
Yes, it will come on its own PR, I believe that we can split.